modules/mono/glue/GodotSharp/GodotSharp/Core/Array.cs

```diff
@@ -15,14 +14,14 @@ namespace Godot.Collections
     /// </summary>
     public sealed class Array : IList, IDisposable
     {
-        internal godot_array NativeValue;
+        public godot_array NativeValue;
```

Comment:
I'm still thinking what to do about this, but for now it needs to be public for marshalling to be possible in the `GodotTools` assembly. In the future this will be needed by game projects too, once we add  source generators for marshalling. I will have to find a proper way to encapsulate this before that.